cmdLine/src/main/scala/topgun/cmdline/FileParser.scala

```diff
@@ -228,9 +227,11 @@ class FileParser(file: File, cmdLine: JfrParseCommandLine, totals: Totals, confi
           val method = frame.getMethod
           if(method ne null) {
             val splitIndex = method.getType.getName.lastIndexOf(".")
+            val packageName = method.getType.getName.substring(0, splitIndex).intern()
```

Comment:
logic on `splitIndex` should be here

e.g val packageName  if (splitIndex == -1)....